frontend/src/css/Index.css

```diff
@@ -0,0 +1,5 @@
+body{
+    margin: 0;
+    height: 100vh;
+    width: 100vw;
+}
```

Comment:
maybe use GlobalStyle from styled-components, so you don't have to mix it up with css files

Reply:
I will give it a try :D